diktat-rules/src/test/resources/test/paragraph1/naming/identifiers/VariableNamingTest.kt

```diff
@@ -10,3 +10,18 @@ private val loWer_VAlue = ""
 private val lower = ""
 private val ValN_x256 = ""
 private val VoiceIP_port = ""
+
+class A {
```

Comment:
We need more tests: how are local variables renamed, how are variables from companion object renamed when used in class (does `findAllVariablesWithUsages` find even them?).